main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:     bike
+//  action 1: tires pop
+bike.popTire();
+//  action 2: stops
+bike.stop();
+//  action 3: accelerates
+bike.accelerate();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:     elevator
+//  action 1: close doors
+elevator.closeDoors();
+//  action 2: calls help
+elevator.connectToEmergencyServices();
+//  action 3: goes to lobby
+elevator.goToLobby()
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:     guitar
+//  action 1: string breaks
+guitar.breakString();
+//  action 2: increase pitch of string
+guitar.increaseStringPitch();
+//  action 3: switch pickout wiring
+guitar.outputPickup();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:     tv
+//  action 1: display different channel
+tv.newChannel();
+//  action 2: play audio
+tv.playAudio();
+//  action 3: connect to wifi
+tv.connectWifi();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:     plane
+//  action 1: consume gas
+place.burnGas();
+//  action 2: descend
+plane.descend();
+//  action 3: landing gear
+plane.extendLandingGear();
```

Comment:
`landing gear` is not an action.
your pseudo-code is correctly expressed.
your plain-english action needs the word `extend` added, as that is the action.
- [ ] revise